pyrgg/functions.py

```diff
@@ -470,12 +487,12 @@ def branch_gen(
                 max_edge - 1)):
             reference_vertices.pop(random_tail_index)
             continue
-        if direct == 2:
+        if not direct:
             if random_tail in used_vertices.keys():
                 used_vertices[random_tail].append(vertex_index)
             else:
                 used_vertices[random_tail] = [vertex_index]
-        if sign == 2:
+        if not sign:
             random_weight = random_unit(min_weight, max_weight)
         else:
             random_weight = sign_gen() * random_unit(min_weight, max_weight)
```

Comment:
Can you change it to
```
if sign:
    random_weight = sign_gen() * random_unit(min_weight, max_weight)
else:
    random_weight = random_unit(min_weight, max_weight)
```
I think it may be more comfortable for other code viewers to get along with code this way.

Reply:
You're right. I inverted that if statement. Thank you for your review.